lib/hideUI.js

```diff
@@ -25,6 +25,10 @@ define(function (require, exports, module) {
 	 * This function merely removes the left side tool bar
 	 */
 	function removeLeftSideToolBar() {
+		//Remove header titles
```

Comment:
Fix this comment to reflect that we're hiding the right-split view details, since we're hijacking it.